aravis/Cargo.toml

```diff
@@ -15,12 +15,22 @@ documentation = "https://docs.rs/aravis"
 [features]
 dox = ["aravis-sys/dox", "nightly"]
 nightly = []
+v0_8_3 = []
+v0_8_6 = ["v0_8_3"]
+v0_8_8 = ["v0_8_6"]
+v0_8_11 = ["v0_8_8"]
+v0_8_15 = ["v0_8_11"]
+v0_8_16 = ["v0_8_15"]
+v0_8_17 = ["v0_8_16"]
+v0_8_19 = ["v0_8_17"]
+v0_8_20 = ["v0_8_19"]
+v0_8_22 = ["v0_8_20"]
```

Comment:
Could you add a changelog entry for this too?

Reply:
Done :+1: